### DISCUSSION TITLE
part-ii/2017-122/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#203] [Question comments]
Part II 2017 Probability and Measure: Paper 4, Section II, J 

https://questions.tripos.org/part-ii/2017-122/

**Comment by eha1234** [comment]
<details><summary>(d) (i)</summary>

Show $\mu = \mu_\theta$ for intervals of the form $[\frac{i}{2^n},\frac{i+1}{2^n})$. 
</details>

<details><summary>(d) (ii)</summary>

Not ergodic. e.g. $A = [0,1/4)$
</details>
